pages/app/orgsettings.tsx

```diff
@@ -0,0 +1,103 @@
+                <button
+                  className="py-2 px-4 w-44 bg-gray-400 text-white rounded-lg hover:bg-gray-600 cursor-pointer transition duration-500 ease-in-out text-sm"
+                  onClick={() => setDeleteModal(false)}
+                >
+                  Cancel
+                </button>
+                <button
+                  className="py-2 px-4 w-44 bg-red-500 text-white rounded-lg hover:bg-red-600 cursor-pointer transition duration-500 ease-in-out text-sm"
+                  onClick={() => ''}
+                >
+                  Delete Organization
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      )}
+      <div className="max-w-6xl container mx-auto flex flex-col items-center justify-center relative">
+        <div className="w-3/4 flex flex-row justify-center items-center mb-24 relative">
+          <div className="w-1/5 mt-1">
+            <button>
+              <ArrowNarrowLeftIcon className=" h-6 w-6" />
+            </button>
+          </div>
+          <label className=" w-4/5 font-semibold text-4xl">{orgName} Settings</label>
+        </div>
+        <div className="max-w-lg relative">
+          <div>
+            <label className="text-lg text-gray-500">Organization Name</label>
+            <input
+              className="rounded-lg h-10 border-gray-300 border block mb-6 mt-2 px-28"
+              value=""
+            ></input>
+            <div className="flex flex-row">
+              <label className="border h-28 w-28 bg-gray-500 rounded-xl">Image</label>
+              <div className="flex flex-col justify-around items-center px-8">
+                <label className="text-lg">Organization Avatar</label>
+                <button
+                  onClick={() => ''}
+                  className="py-2 text-sm w-32 bg-primary text-gray-50 rounded-lg hover:bg-secondary cursor-pointer transition duration-500 ease-in-out"
+                >
+                  Upload Image
+                </button>
+              </div>
+            </div>
+          </div>
+          <div className="mt-6">
+            <label className="text-lg text-gray-500">Organization Members</label>
+            <div className="flex flex-row justify-center items-center">
+              <input
+                className="rounded-lg h-10 border-gray-300 border block mb-6 mt-2 px-7 w-3/4"
+                placeholder="User Email"
+              ></input>
+              <div className="w-1/4 mx-auto">
+                <button className="text-primary text-3xl mb-6 mx-6" onClick={() => ''}>
+                  +
+                </button>
+              </div>
+            </div>
+            <ul className="font-semibold text-lg mb-16">
+              {orgMembers.map((item, index) => (
+                <li key={index} className="m-3 flex flex-row items-center">
+                  <UserCircleIcon className="w-6 h-6 mr-4" />
```

Comment:
Are we using this icon or are we using the users avatar?

Reply:
I don't know what we will use, so I added the icon to have a component in that space. If we are going to use avatars I will change it when the parameters are available.